Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -161,6 +239,45 @@ private DomainModel.Models.Test.Test CreateTest(string testName)
             return test;
         }
 
+        public QuestionAC CreateQuestionAc(bool isSelect, string questionDetails, int categoryId, int id)
+        {
+
+            var QuestionAc = new QuestionAC()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Rename 'QuestionAc' which hides the property with the same name. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1117)